packages/eas-cli/src/project/uploadAccountScopedProjectSourceAsync.ts

```diff
@@ -36,15 +39,7 @@ export async function uploadAccountScopedProjectSourceAsync({
     const projectTarball = await makeProjectTarballAsync(vcsClient);
     projectTarballPath = projectTarball.path;
 
-    if (projectTarball.size > 1024 * 1024 * 100) {
-      Log.warn(
-        `Your project archive is ${formatBytes(
-          projectTarball.size
-        )}. You can reduce its size and the time it takes to upload by excluding files that are unnecessary for the build process in ${chalk.bold(
-          '.easignore'
-        )} file. ${learnMore('https://expo.fyi/eas-build-archive')}`
-      );
-    }
+    maybeWarnAboutProjectTarballSize(projectTarball.size);
 
     if (projectTarball.size > 2 * 1024 * 1024 * 1024) {
       throw new Error('Project archive is too big. Maximum allowed size is 2GB.');
```

Comment:
do we want to move it as well to separate function?

Reply:
👍 
https://github.com/expo/eas-cli/blob/242f949236145b40477bba9a51ad0b9a331ffd34/packages/eas-cli/src/build/utils/repository.ts#L186-L198